releases/unreleased/redirect-to-the-original-url-after-log-in.yml

```diff
@@ -0,0 +1,8 @@
+---
+title: Redirect to the original URL after log in
+category: added
+author: Eva Milán <evamillan@bitergia.com>
```

Comment:
LGTM, works as expected. Please check your author's surname, this will create a new entry in the AUTHORS file :smile: https://github.com/chaoss/grimoirelab-sortinghat/blob/master/AUTHORS